e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -231,10 +231,8 @@ describe('Pricing modal', () => {
         // *Check for contact sales CTA
         cy.get('#contact_sales_quote').contains('Contact Sales for a quote');
 
-        // *Check that enterprise card action button opens LearnMoreTrialModal
-        cy.get('#pricingModal').get('#enterprise').get('#enterprise_action').contains('Try free for 30 days');
-        cy.get('#pricingModal').get('#enterprise').get('#enterprise_action').click();
-        cy.get('.LearnMoreTrialModal').should('exist');
+        // *Check that enterprise card action button shows Try free for 30 days
```

Comment:
Should we expand the tests here to check the differences for self-hosted? The Looking for cloud and that the plan has data restrictions is not here?